sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -118,6 +118,9 @@ class SessionCatalog(
   }
 
   def dropDatabase(db: String, ignoreIfNotExists: Boolean, cascade: Boolean): Unit = {
+    if (db == "default") {
```

Comment:
Now, I think we should issue an exception if users set `CASE_SENSITIVE` to `true` if users use Hive Metastore.